### DISCUSSION TITLE
Test of https://github.com/cedricp/ddt4all/issues/641

### DISCUSSION THREAD

**Discussion by Furtif** [cedricp/ddt4all#841] [General]
![9178ABEF-BDE5-4FA1-AA8E-8A42ED19A6F2](https://github.com/cedricp/ddt4all/assets/11375035/0aee74b8-570a-468e-baf3-2741e6c4a005)
![8FD066E7-C878-4A21-9C3E-44B0C0E8B771](https://github.com/cedricp/ddt4all/assets/11375035/426f3e14-fdc4-4fa3-b116-d052bc87cbf0)
![2B4BB326-2836-4AF2-902C-083C0A064ED2](https://github.com/cedricp/ddt4all/assets/11375035/13f37bca-d03f-4b8d-b87c-4df26c562fd1)
![0566AD5D-C9A7-4F7D-B76F-E916C08DB576](https://github.com/cedricp/ddt4all/assets/11375035/7b238fbc-c10e-419d-b048-4b241f6d578f)
![B394F4C7-80F7-49AB-B01E-D8A82728CE05](https://github.com/cedricp/ddt4all/assets/11375035/03de5295-1dce-4fd8-b607-e2d220118386)

test … all screen ok etc but can not save config … odd …

**Comment by dexter2111** [comment]
@Furtif 
As mentioned, I can set the time, but nothing happens.
Settings can only be changed with the engine running.
Thank you for your time, testing this "problem".
It is a useful function during the winter.
